selene/src/main.rs

```diff
@@ -517,17 +530,23 @@ fn start(mut options: opts::Options) {
                 }
             }
 
-            None => match fs::read_to_string("selene.toml") {
-                Ok(config_contents) => match toml::from_str(&config_contents) {
-                    Ok(config) => (config, None),
-                    Err(error) => {
-                        error!("Config file not in correct format: {}", error);
-                        std::process::exit(1);
-                    }
-                },
+            None => {
+                let read_config_file = |file: &str| -> Option<CheckerConfig<toml::value::Value>> {
+                    let config_contents = fs::read_to_string(file).ok()?;
+                    toml::from_str(&config_contents)
+                        .map_err(|error| {
+                            eprintln!("Error parsing config file {}: {}", file, error);

```

Comment:
Use `error!` for consistent formatting.

Reply:
solved in https://github.com/Zeioth/selene/commit/b4a93e328ebbdefcd336278648952b0974610117